### ISSUE TITLE
Why don't we have TestEngine.AsyncTest similar to QUnit.asyncTest

### ISSUE THREAD

**Issue by chandramuralis** [nikhilk/scriptsharp#59]
I'm trying to create a test case for my class and which executes in asynchronous way, and I couldn't see something similar to <code>QUnit.asyncTest( name, expected, test )</code> function available in <b>QUnit framework</b>. Is there any other alternative function available to achieve the same or could you help us to point to any documentation other than the PhotoDemo sample that came with 0.7.1.0 version?

**Comment by nikhilk** [comment]
Did you see my sample from MIX? It has an example of writing async tests if I remember correctly.

The interesting APIs are:

```
TestEngine.WaitForAsyncCompletion // maps to QUnit.stop
TestEngine.ResumeOnAsyncCompleted // maps to QUnit.start
```

Using these you can start some async work, call Wait, and in the completed handler, resume.

**Comment by chandramuralis** [comment]
will go through that. Thanks.

Actually I tried to watch that Video many times, but the streaming doesn't work in channel9 link, it stops after 2 minutes. Anyhow I'll give it another try today. Thanks

**Comment by chandramuralis** [comment]
At last I downloaded your MIX presentation and watched it completely. I went through the sample there and it worked perfect after that. There is one more thing I wanted to clarify, does Script# Test framework mandates to write our Test functions with the method name starting with "Test". Because, when I wrote a function that doesn't start with "Test", Script# doesn't considered it as a Test method.

By the way great presentation in MIX.
